client/src/components/leftSidebar/topLeftLogoAndTitle.js

```diff
@@ -13,15 +13,14 @@ const DATASET_TITLE_FONT_SIZE = 14;
 @connect((state) => {
   const { corpora_props: corporaProps } = state.config;
   const correctVersion =
-    ["1.0.0", "1.1.0"].indexOf(
-      corporaProps?.version?.["corpora_schema_version"]
-    ) > -1;
+    ["1.0.0", "1.1.0"].indexOf(corporaProps?.version?.corpora_schema_version) >
```

Comment:
Can this be removed or filed to be removed?

Reply:
See #2038